test/markup/handlebars/built-ins.expect.txt

```diff
@@ -6,7 +6,7 @@
 
 </span><span class="hljs-template-tag">{{#<span class="hljs-name"><span class="hljs-builtin-name">each</span></span> test}}</span><span class="xml">abc</span><span class="hljs-template-tag">{{/<span class="hljs-name"><span class="hljs-builtin-name">each</span></span>}}</span><span class="xml">
 
-</span><span class="hljs-template-variable">{{<span class="hljs-builtin-name">lookup</span> abc}}</span><span class="xml">
+</span><span class="hljs-template-variable">{{<span class="hljs-name"><span class="hljs-builtin-name">lookup</span></span> abc}}</span><span class="xml">
 
-</span><span class="hljs-template-variable">{{<span class="hljs-builtin-name">log</span> test}}</span><span class="xml">
+</span><span class="hljs-template-variable">{{<span class="hljs-name"><span class="hljs-builtin-name">log</span></span> test}}</span><span class="xml">
```

Comment:
This is annoying but I'm not sure we should fix it, and perhaps it's technically accurate in any case?

Reply:
I am not sure how to interpret the definition in the [docs](https://highlightjs.readthedocs.io/en/latest/css-classes-reference.html)

>  name | name of an XML tag, the first word in an s-expression
>  builtin-name | s-expression name from the language standard library

When I remove the `name` and try it in the `developer.html`, the tag name (also the `builtin-name` is not bold anymore. Of course, this depends on the theme.

But I am also not sure how to fix it. It is cause by the combination of the `name`-mode for mustache expressions and the `keywords`.

And it happens for block-helpers in the `highlight.js/master` as well:

```handlebars
{{#if test}}true{{/if}}
```

Reply:
I think in a perfect world we'd see only one or the other, not both, but this isn't a big deal. You could even argue this is technically correct.

> When I remove the name and try it in the developer.html, the tag name (also the builtin-name is not bold anymore. Of course, this depends on the theme.

Ah, that makes a slight argument in favor of "it should be both because both styles would apply in cases where they can".

I will make this resolved.